spec/models/request_spec.rb

```diff
@@ -494,4 +494,21 @@
       expect(subject[request_type2].started).to eq(1)
     end
   end
+
+  describe '.get_all_comments' do
```

Comment:
```suggestion
  describe '#get_all_comments' do
```

I was going to say that this is the convention elsewhere in Sequencescape. But it looks like we've got both!

Reply:
'.' is for class methods. '#' for instance methods.

Reply:
in this case, I should use '.' then?

Reply:
The signature self.get_all_comments(request) shows it is a class method. Test for this method by convention will have a '.' prefix.

Reply:
OK, thanks! will change back my initial one.